assets/js/components/common/Mapbox.jsx

```diff
@@ -27,8 +32,13 @@ export default (props) => {
         properties: {
           id: h.hotspot_address,
           name: `${h.hotspot_name}`,
-          description: `description for hotspot ${h.hotspot_address}`,
+          alias: hotspot_alias,
           packetCount: h.packet_count,
+          isFollowed: String(
```

Comment:
maybe you can use orgHotspotsMap to check if a hotspot is followed? instead of a linear time filter function

Reply:
actually doesnt matter since we will paginate followed hotspots list later, nvm

Reply:
oh we are? when we do, will have to change how i check if followed or not then